package/helm-cmd

```diff
@@ -24,7 +24,7 @@ for i in operation*; do
       cp $kustomization kustomization.yaml
     fi
     cat $i | xargs -0 -- echo helm
-    cat $i | xargs -0 -- helm
+    cat $i | xargs -0 -- helm 2>&1 | sed '/WARNING: Kubernetes configuration file is group-readable/d; /WARNING: Kubernetes configuration file is world-readable/d'
```

Comment:
Just curious, if you have tried exploring fixing the KUBECONFIG file permissions since I think that is a better solution than this.

Reply:
I did, and I agree with you, but that sounds like a bigger change that could have unforeseen effects that would be harder to test